.deepsource.toml

```diff
@@ -0,0 +1,11 @@
+version = 1
```

Comment:
We don't have permissions to subscribe this repo to that service. So please remove this file.

Reply:
@ccordoba12 removed `.deepsource.toml` file.
These were some of the changes that I found with DeepSoure https://deepsource.io/gh/pnijhara/python-language-server/issues/?category=recommended

If in case you need any help regarding Deepsource! Ping me up!